src/org/usfirst/frc4904/robot/subsystems/GearIO.java

```diff
@@ -78,4 +82,14 @@ public void setRampState(RampState state) {
 	public void initDefaultCommand() {
 		setDefaultCommand(new Idle(this));
 	}
+
+	@Override
+	public void setOverride(boolean setValue) {
```

Comment:
With setters, just like constructors, the method argument should be the same name as the property - eg. `this.rampOverride = rampOverride`